app/templates/main/error.hbs

```diff
@@ -1,7 +0,0 @@
-{{#if slug}}
```

Comment:
For reference, this is what current prod looks like:
<img width="1317" alt="screen shot 2017-05-31 at 10 48 30 am" src="https://cloud.githubusercontent.com/assets/366944/26623680/be19fb3c-45ee-11e7-9e4c-580bfd24b97d.png">